libkbfs/root_metadata.go

```diff
@@ -90,6 +90,10 @@ func (p PrivateMetadata) ChangesBlockInfo() BlockInfo {
 	return p.cachedChanges.Info
 }
 
+// ExtraMetadata is a per-version blob of extra metadata which may exist outside of the
+// given metadata block, e.g. key bundles for post-v2 metadata.
+type ExtraMetadata interface{}
```

Comment:
The problem with using `interface{}` is that type errors like passing in some random argument for an `ExtraMetadata` argument won't be caught at compile time. Maybe it's worth adding a method, say, something like `GetMetadataVersion()` to guard against that?